api-spec/payment-requests-api.yaml

```diff
@@ -696,4 +703,10 @@ components:
         description:
           description: Payment notice description
           type: string
-          maxLength: 140 # as paymentDescription for paForNode
\ No newline at end of file
+          maxLength: 140 # as paymentDescription for paForNode
+    ClientId:
+      type: string
+      description: Enumerations of client ids set by APIM
+      enum:
+        - WISP_REDIRECT
+        - CHECKOUT_CART
```

Comment:
would the `CHECKOUT` client be missing?

Reply:
I thought we wouldn't expect requests with CHECKOUT as clientId for this specific API (post-carts). Is that correct?

Reply:
the "type" in openapi is generic, not dedicated to post carts.
the idea is that in the future it could be useful to have the clientId also for the `GET payment-requests` (to monitor), so either we rename the definition name with some reference to the `POST carts` or we insert all the admissible clients for the payment-request microservice.

Reply:
Enum updated in [beb61a4](https://github.com/pagopa/pagopa-ecommerce-payment-requests-service/pull/111/commits/beb61a4f4979a01a5ec62a6b58cdad28a8015c60)